dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/dashboard/DashboardExportSearchResultsPageTest.java

```diff
@@ -25,24 +25,20 @@
 @ExtendWith(MockitoExtension.class)
 @MockitoSettings(strictness = LENIENT)
 public class DashboardExportSearchResultsPageTest {
-
-    @InjectMocks
-    private DashboardExportSearchResultsPage page;
-
     @Mock
     private DataverseSession session;
     @Mock
     private SystemConfig systemConfig;
     @Mock
     private DatasetFieldTypeRepository datasetFiledTypeRepo;
     @Mock
-    private PermissionsWrapper permissionsWrapper;
```

Comment:
You still need a `NavigationWrapper` mock.

Reply:
I dont need it since I don't test DashboardExportSearchResultsPage.verifyAccess() method, and I don't intend to for now.

Reply:
But you're purposefully breaking the constructor interface by providing null and you're still creating a `SystemConfig` mock even though it doesn't seem necessary. I mean, it's not very consistent. Tests are in a way much like production code.

Reply:
fixed